concent_api/core/admin.py

```diff
@@ -1,20 +1,21 @@
 from django.contrib import admin
 
-from common.admin    import ModelAdminReadOnlyMixin
-from .models        import PendingResponse
-from .models        import StoredMessage
-from .models        import Subtask
+from common.admin import ModelAdminReadOnlyMixin
+from common.helpers import get_current_utc_timestamp
+from .models import PendingResponse
+from .models import StoredMessage
+from .models import Subtask
 
 
-class ActivePassiveStateFilter(admin.SimpleListFilter):
-
-    title          = 'Active/passive state'
+class ActivePassiveDownloadsStateFilter(admin.SimpleListFilter):
+    title = 'Active/passive/downloads state'
     parameter_name = 'active_passive_state'
 
     def lookups(self, request, model_admin):
         return (
-            ('active',  'Active'),
+            ('active', 'Active'),
             ('passive', 'Passive'),
+            ('active_downloads', 'With active downloads')
```

Comment:
Do we want it as separate option or we would like to have `active` + `active_downloads`